File: kelvin.js
// The code below stores the value of 293 into a constant variable called kelvin
const kelvin = 345;
// The code below converts kelvin to celsius by subtracting 273 from kelvin
const celsius = kelvin - 273;
// The line of code below converts celsius into fahrenheit
let fahrenheit = celsius * (9/5) + 32;
// The line of code below uses the math.floor method to round up temp in fahrenheit with decimal to the nearest whole number.
fahrenheit = Math.floor(fahrenheit);
console.log(`The temperature is ${fahrenheit} degrees fahrenheit`);
console.log(`${kelvin} Kelvin is about ${fahrenheit} degrees Fahrenheit.`);
console.log(`${kelvin} Kelvin is about ${celsius} degrees Celsius.`);
// convert Celsius to Newton
let newton = celsius * (33 / 100);
// round down using math.floor method
newton = Math.floor(newton);
// print the value of newton to the console
console.log(`The temperature is ${newton} degrees Newton`);
// kelvin's value in newton
console.log(`${kelvin} Kelvin is about ${newton} degrees Newton.`)

File: kelvin.py
3# Ask the user for kelvin's input
kelvin = input("Enter the value of Kelvin here ")
kelvin = int(kelvin)
#Get the value of celsius by deducting 273 from kelvin
celsius = kelvin - 273
# Determine the value of fahrenheit
fahrenheit = celsius * (9/5) + 32
# Convert fahrenheit to integer value
fahrenheit = int(fahrenheit)
# Determine the newton
newton = celsius * (33/100)
# convert newton to integer
newton = int(newton)
# Print al the values
# print temperature in fahrenheit
print(f"The temperature is {fahrenheit} degrees Fahrenheit.")
# convert kelvin to fahrenheit
print(f"{kelvin} Kelvin is about {fahrenheit} degrees Fahrenheit")
# print temperature in celsius
print(f"The temperature is {celsius} degrees Celsius")
# Convert kelvin to celsius
print(f"{kelvin} Kelvin is about {celsius} degrees Celsius")
# print temperature in newton
print(f"The temperature is {newton} degrees Newton")
# convert kelvin to newton
print(f"{kelvin} Kelvin is about {newton} degrees Newton")
